lua/telescope/finders/async_job_finder.lua

```diff
@@ -0,0 +1,73 @@
+local log = require('telescope.log')
+local Job = require('plenary.job')
+
+local async_lib = require('plenary.async_lib')
+local async = async_lib.async
+-- local await = async_lib.await
+local void = async_lib.void
+
+local make_entry = require('telescope.make_entry')
+
+return function(opts)
+  local entry_maker = opts.entry_maker or make_entry.gen_from_string()
+  local fn_command = function(prompt)
+    local command_list = opts.command_generator(prompt)
+    if command_list == nil then
+      return nil
+    end
+
+    local command = table.remove(command_list, 1)
+
+    return {
+      command = command,
+      args = command_list,
+    }
+  end
+
+  local job
+  return setmetatable({
+    close = function() end,
+  }, {
+    __call = void(async(function(prompt, process_result, process_complete)
+      print("are we callin anything?", job)
```

Comment:
debug?

Reply:
This code isn't called anywhere at the moment. async job finder is not enabled yet.